scaffolding-chef-infra/lib/linux/client-chunk.rb

```diff
@@ -0,0 +1,13 @@
+cfg_env_path_prefix = {{cfg.env_path_prefix}}
```

Comment:
Hi @afiune 

I think you're right that this would be stronger with single quotes, as ruby will treat this as a string literal. Thanks for this feedback.

Reply:
Stronger? I mean this is ruby, and ruby, as far as I know, ruby can’t understand strings without quotes.

Simple test with quotes:
```
irb(main):002:0> cfg_env_path_prefix ='abc'
=> "abc"
```
Simple test **WITHOUT** quotes:
```
irb(main):001:0> cfg_env_path_prefix = abc
Traceback (most recent call last):
        2: from /usr/local/bin/irb:11:in `<main>'
        1: from (irb):1
NameError (undefined local variable or method `abc' for main:Object)
```

Reply:
Thank you! @afiune I've got a PR open to fix the bug :) #141